pkg/bootstrap/docker/up.go

```diff
@@ -423,10 +424,19 @@ func getDockerClient(out io.Writer, dockerMachine string, canStartDockerMachine
 		return dockerClient, engineAPIClient, nil
 	}
 
+	dockerTLSVerify := os.Getenv("DOCKER_TLS_VERIFY")
+	dockerCertPath := os.Getenv("DOCKER_CERT_PATH")
+	if len(dockerTLSVerify) > 0 && len(dockerCertPath) == 0 {
+		home := homedir.Get()
```

Comment:
I think using
```
dockerCertPath = cliconfig.ConfigDir()
os.SetEnv("DOCKER_CERT_PATH", dockerCertPath)
```

would be preferable here as it should additionally handle the DOCKER_CONFIG env var (thanks Docker!) and keep us closer to the Docker client library for when we remove the fsouza client one day.